helm/reporting-agent-chart/values.yaml

```diff
@@ -1,14 +1,14 @@
 image: quay.io/giantswarm/reporting-agent
 image_tag: latest
-index: agent-test1
+index: gs-agent-test
```

Comment:
This should not be `gs-agent-test-*` like in here: https://github.com/giantswarm/reporting-processor/blob/ad74ebe4230d2a0a2e3222d4ec364db078c98600/helm/reporting-processor-chart/values.yaml#L4